rbac/management/role/definer.py

```diff
@@ -84,3 +84,36 @@ def seed_roles(tenant, update=False):
             role_list = data.get("roles")
             _update_or_create_roles(tenant, role_list, update)
     return tenant
+
+
+def seed_permissions(tenant):
+    """For a tenant update or create defined permissions."""
+    permission_directory = os.path.join(settings.BASE_DIR, "management", "role", "permissions")
+    permission_files = [
+        f
+        for f in os.listdir(permission_directory)
+        if os.path.isfile(os.path.join(permission_directory, f)) and f.endswith(".json")
+    ]
+    # current_permission_ids = set()
+
+    with tenant_context(tenant):
```

Comment:
I think we'll want to use `with transaction.atomic():` here?

Reply:
Ah ha, good idea 👍